intelmq/etc/harmonization.conf

```diff
@@ -197,6 +197,10 @@
             "regex": "^[a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[0-9a-z]{12}$",
             "type": "LowercaseString"
         },
+        "output": {
```

Comment:
"output" is a very generic name for the field.
On the input side we have a field called "raw" .
I assume what you want here is a field called "transformed" or similar?